Makefile

```diff
@@ -359,7 +359,8 @@ go-install: $(INSTALL_TARGETS)
 .PHONY: clean
 clean:
 ## clean: Clean the cache and test caches
-	go clean -n -r --cache --testcache $(PROJECT)/...
+	go clean -r --cache --testcache
+	go clean $(PROJECT)/...
```

Comment:
clearly nobody has been using clean, given that '-n' is just print what you would do, and "-cache cannot be used with package arguments"

However, I think the actual desire is:
```
go clean --cache --testcache
go clean -r $(PROJECT)/...
```
(at least I don't think recursion makes sense for cleaning the cache)

Reply:
I use it, but put in an alias to work around it. Finally got motivated to fix it. I did wonder sometimes why things seemed off.